src/Init/Tactics.lean

```diff
@@ -39,8 +39,11 @@ be a `let` or function type.
 syntax (name := intro) "intro" notFollowedBy("|") (ppSpace colGt term:max)* : tactic
 
 /--
-`intros x...` behaves like `intro x...`, but then keeps introducing (anonymous)
-hypotheses until goal is not of a function type.
+Introduces zero or more hypotheses, optionally naming them.
+- `intros` is equivalent to repeatedly applying `intro`,
+  introducing anonymous variables so long as the goal is of a function type.
```

Comment:
Will all readers be comfortable thinking of implication as a function type? `intros` is one of the first tactics that one learns, after all.

```suggestion
  introducing anonymous variables so long as the goal is an implication or a function.
```

Reply:
Nice point!

Reply:
I think we want to be careful that we don't accidentally suggest that these are different concepts, but it's a good point that we should point this out. I've made an attempt at wording something, and I've also added examples to illustrate the scope of this tactic.